saleor/checkout/calculations.py

```diff
@@ -110,24 +146,22 @@ def checkout_line_total(
     checkout_info: "CheckoutInfo",
     lines: Iterable["CheckoutLineInfo"],
     checkout_line_info: "CheckoutLineInfo",
+    address: Optional["Address"] = None,
```

Comment:
Do we need it? Previously we were taking an address from checkout_info, which looked like an ok solution.